### PR TITLE
Cyber organs are inedible

### LINKED ISSUE #2001: Cybernetic organs are edible
Issue reported from  Round ID: 4489 (AuStation 13 [AUSTRALIA/OCEANIA])

Reporting client version: 513.1526

While trying to put cybernetic organs into myself, I bit them so I couldn't implant them anymore. I don't think they are supposed to be edible.

### PR DESCRIPTION
**Pull request by totallynotkatfood**
## About The Pull Request

title

## Why It's Good For The Game

Resolves #2001 
Also makes cyber organs visible on medscanners

## Changelog
:cl:
fix: Cybernetics are now inedible
/:cl:

### CODE REVIEW

**Comment by coolsurf6** [comment]
Everything is edible if you put enough effort.

File: code/modules/surgery/organs/liver.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@
 /obj/item/organ/liver/on_life()
 	if(HAS_TRAIT(owner, TRAIT_NOMETABOLISM))
 		return
-		
+
```
Comment (TheFakeElon) [inline code/modules/surgery/organs/liver.dm:28]: whitespace

File: code/modules/surgery/organs/liver.dm [hunk 2]
```diff
@@ -93,6 +93,7 @@
 	icon_state = "liver-c"
 	desc = "An electronic device designed to mimic the functions of a human liver. Handles toxins slightly better than an organic liver."
 	organ_flags = ORGAN_SYNTHETIC
+	status = ORGAN_ROBOTIC //austation cybernetics are not edible
```
Comment (TheFakeElon) [inline code/modules/surgery/organs/liver.dm:96]: Modularise

File: code/modules/surgery/organs/lungs.dm [hunk 3]
```diff
@@ -454,6 +454,7 @@
 	desc = "A cybernetic version of the lungs found in traditional humanoid entities. Allows for greater intakes of oxygen than organic lungs, requiring slightly less pressure."
 	icon_state = "lungs-c"
 	organ_flags = ORGAN_SYNTHETIC
+	status = ORGAN_ROBOTIC //austation cybernetics are not edible
```
Comment (TheFakeElon) [inline code/modules/surgery/organs/lungs.dm:457]: Modularise

File: code/modules/surgery/organs/heart.dm [hunk 4]
```diff
@@ -175,6 +175,7 @@
 	desc = "An electronic device designed to mimic the functions of an organic human heart. Also holds an emergency dose of epinephrine, used automatically after facing severe trauma."
 	icon_state = "heart-c"
 	organ_flags = ORGAN_SYNTHETIC
+	status = ORGAN_ROBOTIC //austation cybernetics are not edible
```
Comment (TheFakeElon) [inline code/modules/surgery/organs/heart.dm:178]: Modularise

**Comment by MCterra10** [comment]
please PR this upstream to beestation. if it's closed, I will merge it here.

**Comment by totallynotkatfood** [comment]
upstream merged, closing pr

**Comment by MCterra10** [comment]
Merged upstream. Closing.

**Pull request closed without merging**